templates/shared/runtime/bin/setup-local-disks

```diff
@@ -95,11 +110,13 @@ EOF
   done
 
   ## Check if /var/log/pods has been bind mounted and make sure kubelet is stopped
-  if [[ "$(systemctl is-active var-log-pods.mount)" != "active" ]]; then
-    if systemctl is-active "kubelet" > /dev/null 2>&1; then
-      prev_running+=" ${unit}"
+  if [[ "${BIND_VAR_LOG_PODS}" == "true" ]]; then
+    if [[ "$(systemctl is-active var-log-pods.mount)" != "active" ]]; then
+      if systemctl is-active "kubelet" > /dev/null 2>&1; then
+        prev_running+=" ${unit}"
```

Comment:
i think this `${unit}` is leaked from the previous loop
```suggestion
        prev_running+=" kubelet"
```

Reply:
I think you are right